cpp/include/cuml/ensemble/randomforest.hpp

```diff
@@ -121,6 +121,12 @@ void build_treelite_forest(ModelHandle* model,
 
 std::vector<unsigned char> save_model(ModelHandle model);
 
+std::vector<unsigned char> concatenate_trees(
+  const cumlHandle& handle, std::vector<ModelHandle*> treelite_handles);
```

Comment:
`std::vector<ModelHandle>`. `ModelHandle` is a typedef of `void*`, so it is already a pointer (actually `tl::Model*`).